src/target/stm32l4.c

```diff
@@ -109,19 +114,20 @@ static int stm32l4_flash_write(struct target_flash *f,
 #define OR_DBANK 		(1 << 22)
 
 #define DBGMCU_CR(dbgmcureg)	(dbgmcureg + 0x04)
-#define DBGMCU_CR_DBG_SLEEP		(0x1U << 0U)
```

Comment:
Whitespace changes

Reply:
But...they are misaligned, why keep them that way?

Reply:
Style versus functional change